h2/src/main/org/h2/message/DbException.java

```diff
@@ -342,6 +341,42 @@ public static DbException convertIOException(IOException e, String message) {
         return get(ErrorCode.IO_EXCEPTION_2, e, e.toString(), message);
     }
 
+    /**
+     * Gets the SQL exception object for a specific error code.
+     *
+     * @param errorCode the error code
+     * @return the SQLException object
+     */
+    public static SQLException getSQLException(int errorCode)
+    {
+        return getJdbcSQLException(errorCode, null, new String[0]);
+    }
+
+    /**
+     * Gets the SQL exception object for a specific error code.
+     *
+     * @param errorCode the error code
+     * @param p1 the first parameter of the message
+     * @return the SQLException object
+     */
+    public static SQLException getSQLException(int errorCode, String p1)
+    {
+        return getJdbcSQLException(errorCode, null, new String[] { p1 });
+    }
+
+    /**
+     * Create the SQL exception object for a specific error code.
+     *
+     * @param errorCode the error code
+     * @param cause the cause of the exception
+     * @param params the list of parameters of the message
+     * @return the exception
+     */
+    public static SQLException getSQLException(int errorCode, Throwable cause, String... params)
+    {
+        return getJdbcSQLException(errorCode, cause, params);
```

Comment:
I don't understand why this wrapper method in needed. `getJdbcSQLException(int, Throwable, String...)` can be called directly if its visibility will be raised to public.

Also I think that all methods should have the same name either `getJdbcSQLException` or shorter `getSQLException`.